site-src/geps/gep-713.md

```diff
@@ -38,10 +48,116 @@ portable.
+object is in both cases a superset of its settings and the settings of a wrapped
+object (or objects). “Meta” here is used in its Greek sense of “more comprehensive”
+or “transcending”, and “resource” rather than “object” because “metaresource”
+is more pronounceable than “metaobject”. Additionally, a single word is better
+than a phrase like “wrapper object” or “wrapper resource” overall, although both
+of those terms are effectively synonymous with “metaresource”.
+
+All ReferenceGrants are metaresources, and all objects that use Policy Attachment
+are metaresources, but metaresources can be more than just those two things.
+
+### Targeted metaresources
+
+ReferenceGrant is an example of a targeted metaresource. It’s tightly bound to a
+particular Kind within a single namespace, and only modifies the behavior of the
+objects that match its binding (by allowing references to them from outside their
+namespace).
+
+We’ve been discussing the idea of having a new object that holds extra capabilities
+for a backend Service (something like BackendCapabilites or similar, name TBD),
+as a way to implement GEP-1282 (Backend Properties). A new resource here that
+referenced a single service and had extra properties that we can’t put into the
+Service object is also an example of a targeted metaresource. The intent is to
+augment the behavior of the bound Service object with the structured data in the
+Spec of BackendCapabilities, since it’s not easy to make changes to a GA upstream
+object like Service. (And arguably, Service is already a pretty overloaded
+resource, so there will need to be good justification for any changes).
+
+With these two examples in mind, here are some guidelines for when to consider
+using a targeted metaresource:
+* The number or scope of objects to be modified is limited or singular. Targeted
+  metaresources should target one specific object (preferred), or a tightly-scoped
+  set of objects (like ReferenceGrant).
+* The modifications to be made to the objects don’t have any transitive information -
+  that is, the modifications only affect the single object that the targeted
+  metaresource is bound to, and don’t have ramifications that flow beyond that
+  object. ReferenceGrant and BackendCapabilities are good examples here because
+  they don’t expect the metaresource to modify the behavior of more than one
+  object. (Even in the ReferenceGrant case, the ability to set a Kind rather
+  than an individual object is syntactic sugar for having to prevent needing to
+  have lots of ReferenceGrants if you have lots of wrapped objects).
+* In terms of status, it should be reasonably easy for a user to understand that
+  everything is working - basically, as long as the targeted object exists, and
+  the modifications are valid, the metaresource is valid, and this should be
+  straightforward to communicate in one or two Conditions. Note that at the time
+  of writing, this is *not* completed.
```

Comment:
> straightforward to communicate in one or two Conditions.

On the policy or the targeted object?

How would you also signal that the policy is still being actively worked on? In other resources we have `generation/observedGeneration` pairing.

Reply:
This part is still TBD, and needs to be finished, yes.